.github/actions/docker-meta/action.yaml

```diff
@@ -0,0 +1,82 @@
+name: 'Docker meta'
+inputs:
+  profile:
+    required: true
+    type: string
+  registry:
+    required: true
+    type: string
+  arch:
+    required: true
+    type: string
+  otp:
+    required: true
+    type: string
+  elixir:
+    required: false
+    type: string
+    default: ''
+  build_elixir:
+    required: true
+    type: string
+  builder_base:
+    required: true
+    type: string
+  owner:
+    required: true
+    type: string
+  docker_tags:
+    required: true
+    type: string
+
+outputs:
+  emqx_name:
+    description: "EMQX name"
+    value: ${{ steps.pre-meta.outputs.emqx_name }}
+  version:
+    description: "docker image version"
+    value: ${{ steps.meta.outputs.version }}
+  tags:
+    description: "docker image tags"
+    value: ${{ steps.meta.outputs.tags }}
+  labels:
+    description: "docker image labels"
+    value: ${{ steps.meta.outputs.labels }}
+
+runs:
+  using: composite
+  steps:
+    - name: prepare for docker/metadata-action
+      id: pre-meta
+      shell: bash
+      run: |
+        emqx_name=${{ inputs.profile }}
+        img_suffix=${{ inputs.arch }}
+        img_labels="org.opencontainers.image.otp.version=${{ inputs.otp }}"
+        if [ -n ${{ inputs.elixir }} ]; then
```

Comment:
Maybe should be `-z`?  Or should use `[[ ... ]]`?

Reply:
I want this condition to work when elixir version is not empty

Reply:
I did some quick tests, it seems that either option is needed to have that behavior:

```bash
$ [ -n ${X} ] && echo yep || echo nope
yep
$ X=''
$ [ -n ${X} ] && echo yep || echo nope
yep
$ X='1.13'
$ [ -n ${X} ] && echo yep || echo nope
yep
$ [[ -n ${X} ]] && echo yep || echo nope
yep
$ X=''
$ [[ -n ${X} ]] && echo yep || echo nope
nope
$ [ -z ${X} ] && echo yep || echo nope
yep
$ X='1.13'
$ [ -z ${X} ] && echo yep || echo nope
nope
```

Reply:
Variable must be in quotes (added in the current version)
```
bash-3.2$ X=''
bash-3.2$ [ -n ${X} ] && echo yep || echo nope
yep
bash-3.2$ X=''
bash-3.2$ [ -n "${X}" ] && echo yep || echo nope
nope
bash-3.2$ X='x'
bash-3.2$ [ -n "${X}" ] && echo yep || echo nope
yep
```

Reply:
You're right, with quotes it works!  Ignore me.  :smiley_cat: